src/InstagramClone.Api/Startup.cs

```diff
@@ -0,0 +1,47 @@
+﻿using InstagramClone.Api.Database;
+using Microsoft.EntityFrameworkCore;
+
+namespace InstagramClone.Api
+{
+    public static class Startup
+    {
+        public static WebApplication InitialIzeApp(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+            ConfigureService(builder);
+            var app = builder.Build();
+            Configure(app);
+            return app;
+        }
+
+        private static void ConfigureService(WebApplicationBuilder builder)
+        {
+            
+            builder.Services.AddDbContext<InstagramCloneDbContext>(options =>
+            {
+                options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnectionString"));
+
+               
+            });
+            builder.Services.AddEndpointsApiExplorer();
+            builder.Services.AddControllers();
```

Comment:
what do these lines do?

Reply:
I honestly got excited and was trying  different methods, I forgot to delete them before pushing.

Reply:
👌